### DISCUSSION TITLE
How do you detect the output format from the jupyter engine?

### DISCUSSION THREAD

**Discussion by bhogan-mitre** [quarto-dev/quarto-cli#8456] [Q&A] [answered]
### Description

With the knitr engine, it is very helpful to be able to check the output format and conditionally adjust the display of some cells based on that.  e.g. `knitr::is_html_output()` and `knitr::pandoc_to()`

Is there a way to get that info when running with the jupyter engine?  

I've read about Conditional Content, but it seems from the [docs](https://quarto.org/docs/authoring/conditional.html) that is only applicable to text or non-executable code blocks.  If there was a comparable, easy way to base `eval` and `include` on a condition from jupyter that would help with quarto jupyter parity with rmarkdown.

**Comment by cderv** [comment, marked as answer]
This is still a feature to implement and track in there: 
* https://github.com/quarto-dev/quarto-cli/issues/2216

To be clear, it works with knitr engine in Quarto because we inherits any knitr behavior, and knitr knows about Pandoc. So the information is passed. 

For Jupyter engine, we need to add a mechanism to pass the information.

**Reply by bhogan-mitre** [reply]
Understood, thanks.  I'll follow along on that issue.
